.gitignore

```diff
@@ -26,3 +26,4 @@ classical/droidippy/games/*.txt
 test_output_maps
 generator/*.go
 generator/*map.svg
+variants/ancientmediterranean/svg/flags + units.txt
```

Comment:
If you want to add a new type of file to this then please make it a bit more generic.  e.g. Call the file dev-info.txt or something.